target/linux/ramips/image/rt305x.mk

```diff
@@ -910,6 +910,13 @@ define Device/kn
 endef
 TARGET_DEVICES += kn
 
+define Device/zyxel_keenetic-start
+  DTS := kn_st
```

Comment:
I'm not sure about this one as there's already a bunch of `kn_` filenames already, but I personally find it very messy to name the filenames this way, so I would probably rename the DTS filename to `zyxel-keenetic-start.dts` as well. But this just my personal nitpick.

Reply:
When I started to adding DTS for this device, I took as an example the DTS files of other Keenetic series boards. This is why I named DTS file like this. But yeah, it's realy messy.